docs/source/tutorial/commercial_paper.md

```diff
@@ -140,11 +140,11 @@ be members of.
 
 ![commercialpaper.network](./commercial_paper.diagram.testnet.png)
 *The Fabric test network is comprised of two peer organizations, Org1 and Org2,
-each with one peer and its ledger database, an ordering node. Each of these
-components runs as a Docker container.*
+each with one peer and its ledger state database, and an ordering service node.
+Each of these components runs as a Docker container.*
 
-The two peers, the peer [ledgers](../ledger/ledger.html#world-state-database-options), the
-orderer and the CA each run in the their own Docker container. In production
+The two peers, their state databases [ledgers](../ledger/ledger.html#world-state-database-options), the
```

Comment:
Their state databases ledgers? Did you mean to say state database ledgers? Or maybe just say state databases, remove ledgers and move the link text.

Reply:
done